linkedincopy/mainpage.html

```diff
@@ -0,0 +1,150 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Feed | LinkedIn</title>
+    <link rel="stylesheet" href="mainpage.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.1.3/dist/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.3.0/font/bootstrap-icons.css">
+    <link href="https://getbootstrap.com/docs/5.3/assets/css/docs.css" rel="stylesheet">
+
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/js/bootstrap.bundle.min.js"></script>
+</head>
+<body>
+<div class="navbar1">
+    <form class="d-flex" role="search">
+        <a href="#">
+            <img sizes="" src="./src/pictures/LinkedIn_icon.svg" alt="logo"/>
+        </a>
+        <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+    </form>
+    <nav class="shift-menu">
+        <ul>
+            <li>
+                <div>
```

Comment:
looks like you don't need additional div here(and the same li elements as well)

Reply:
fixed